### PR TITLE
Changes "Nanotransen" to "Nanotrasen" and fixes typos

### PR DESCRIPTION
**Pull request by BestName**
There are 444 instances of "Nanotrasen" in the code and 11 instances of "Nanotransen" so clearly "Nanotrasen" is the correct spelling.
Also fixes a few typos.

### CODE REVIEW

File: code/game/objects/items/mountable_frames/paintings.dm [hunk 1]
```diff
@@ -120,13 +120,13 @@ var/global/list/available_paintings = list(
...
 			name = "\improper Wai-Blu"
-			desc = "Faithfully Serving NanoTransen during her shift, gladly serving YOU after."
+			desc = "Faithfully Serving Nanotrasen during her shift, gladly serving YOU after."
 		if("Kate")
 			name = "\improper Cindy Kate"
 			desc = "Through the carnage and bloodshed she's gunning for you, champ."
 		if("daddy")
 			name = "\improper I <3 Daddy!"
-			desc = "'NanoTransen respects the right for all associates and their families to be able to express their indivuality though many media. However, soliciting NanoTransen related merchandise without proper warrant is strickly prohibited. Luckly for you, you can now own your very own contraband NanoTransen merch without the threat of *REDACTED*!'"
+			desc = "'Nanotrasen respects the right for all associates and their families to be able to express their indivuality though many media. However, soliciting Nanotrasen related merchandise without proper warrant is strickly prohibited. Luckly for you, you can now own your very own contraband Nanotrasen merch without the threat of *REDACTED*!'"
```
Comment (Exxion) [inline code/game/objects/items/mountable_frames/paintings.dm]: > strickly prohibited

File: code/game/objects/items/mountable_frames/paintings.dm [hunk 2]
```diff
@@ -256,13 +256,13 @@ var/global/list/available_paintings = list(
...
```
Comment (Exxion) [inline code/game/objects/items/mountable_frames/paintings.dm]: > strickly

Comment (BestName) [inline code/game/objects/items/mountable_frames/paintings.dm]: pls

**Comment by ghost** [comment]
:+1:

**Comment by clusterfack** [comment]
Wow everybody knows its nanotransen

**Comment by ihadtoregisterforthis** [comment]
[autism] or [Muh Immulsions]

File: code/game/jobs/job/civilian.dm [hunk 3]
```diff
@@ -383,7 +383,7 @@
 	idtype = /obj/item/weapon/card/id/centcom
 	total_positions = 2
 	spawn_positions = 2
-	supervisors = "NanoTransen Law, CentComm Officals, and the stations captain."
+	supervisors = "Nanotrasen Law, CentComm Officals, and the stations captain."
```
Comment (Kurfursten) [inline code/game/jobs/job/civilian.dm]: station's

File: code/modules/events/money_spam.dm [hunk 4]
```diff
@@ -55,7 +55,7 @@
 					"You have (1) new message!", \
 					"You have (2) new profile views!")
 				if(3)
-					sender = pick("Galactic Payments Association", "Better Business Bureau", "Tau Ceti E-Payments", "NAnoTransen Finance Deparmtent", "Luxury Replicas")
+					sender = pick("Galactic Payments Association", "Better Business Bureau", "Tau Ceti E-Payments", "Nanotrasen Finance Deparmtent", "Luxury Replicas")
```
Comment (Kurfursten) [inline code/modules/events/money_spam.dm]: Department

File: code/game/jobs/job/civilian.dm (hunk 3 above)
Comment (BestName) [inline code/game/jobs/job/civilian.dm]: pls m8, I just wanted to change nanotransen

Comment (Kurfursten) [inline code/game/jobs/job/civilian.dm]: Such is the curse of any fix. You end up fixing 4x as many **oldcoder** mistakes as you do your own. Just ask @PJB3005. Every time I refactor something I end up having to delete trailing returns 400 lines up in the file away from where I made the change.

File: code/modules/events/money_spam.dm (hunk 4 above)
Comment (BestName) [inline code/modules/events/money_spam.dm]: STOP!

**Pull request merged**